exploration/percent-format.md

```diff
@@ -58,15 +58,47 @@ use one of the existing number-formatting functions `:number` and `:integer` wit
 or use the proposed _optional_ function `:unit` with an option `unit=percent`.
 Combinations of these approached might also be used.
 
+### Unit Scaling
+
+There is a difference between _input_ scaling and _output_ scaling in `MeasureFormat`,
+which is the model for the `:unit` function in Unicode MessageFormat.
+
+For example, an input of <3.5, `meter`> with `meter` as the output unit doesn't scale.
+
+If one supplies <0.35 `percent`> as the input and the output unit were `percent`, 
+`MeasureFormat` would format as 0.35%. 
+Just like `meter` ==> `meter` doesn't scale.
+
+However, if one supplies a different input unit, then percent does scale 
+(just like `meter` ==> `foot`). 
+The base unit is for such dimensionless units is 'part'.
+In MF, a bare number literal, such as `.local $foo = {35}`
+or an implementation-specific number type (such as an `int` in Java)
+might be considered to use the input unit of `part`
+unless we specified that the `percent` unit value or `:percent` function overrode the `part` unit with `percent`.
```

Comment:
Does `MeasureFormat` or any other unit formatter implementation ever make such an assumption, of having a numerical input value not match the formatter's output units?

Reply:
I don't know about ICU, but CLDR does appear to treat `part` as a [unit](https://github.com/unicode-org/cldr/blob/faa2818490a34a2288f307720ab0abdab8c91ff7/common/validity/unit.xml#L200).

Reply:
Not exactly. There are various measures (such as `parts-per-million`, which is a flavor of `concentr-part` \[concentration\]) that have "parts", but not a measurement `part` all by itself. ICU represents the various measures in `MeasureUnit` and it doesn't have a standalone `PART` member.

Which is beside the point. The percent format in `NumberFormatter` is unscaled when working on a raw number.